packages/docs/src/locales/en/en.json

```diff
@@ -661,16 +661,18 @@
         "canBeFocused": "If false, then input can not be focused using mouse or keyboard."
```

Comment:
ampm true should make selection 12h I assume?

![image](https://user-images.githubusercontent.com/5394573/147934914-f8f606c5-8069-4c3d-aa85-dfaf7b3ca1f0.png)

Reply:
fixed here https://github.com/epicmaxco/vuestic-ui/pull/1394/files#diff-e33525807faa3f844a0b7917a532108e02e8d0a01f85b11e5441ebe7b6789539R110, issue was caused  because computed value wrapped in another computed